tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
I mean a version of the runtime that supports exactly the helper functions that we actually use, with improvements for things like IE8 compatibility.

This is what we've got so far (mostly written by @dgreensp): https://github.com/meteor/react-packages/blob/master/babel-runtime/babel-runtime.js